datasetinsights/stats/visualization/plots.py

```diff
@@ -78,7 +78,7 @@ def _process_label(bbox, label_mappings=None):
         return label
 
 
-def plot_bboxes3d(image, bboxes, projection, colors=None):
+def plot_bboxes3d(image, bboxes, projection, colors=None, orthographic=False):
```

Comment:
We don't have unittests for this method. Can you add it in `test_visual.py`?

Reply:
Yeah I'll do that aswell